extensions/amp-story/1.0/_locales/default.json

```
@@ -38,5 +38,6 @@
   "84": {"string": "SCORE:"},
   "85": {"string": "Pause"},
   "86": {"string": "Play"},
-  "89": {"string": "Your response will be sent to"}
+  "89": {"string": "Your response will be sent to"},
+  "90": {"string": "Move device to explore"}
```

```suggestion
  "96": {"string": "Move device to explore"}
```

Could you update this string to reflect the appropriate key? I think we forgot to add a few here.